sirepo/pkcli/job_supervisor.py

```diff
@@ -59,6 +62,10 @@ def default_command():
         websocket_ping_interval=sirepo.job.cfg.ping_interval_secs,
         websocket_ping_timeout=sirepo.job.cfg.ping_timeout_secs,
     )
+    if cfg.debug:
+        for f in sirepo.util.files_to_watch_for_reload('jinja', 'json', 'py'):
+            tornado.autoreload.watch(f)
```

Comment:
Watching for jinja changes may not be necessary, as they influence output but no logic. I don't think reloads are necessary.

Reply:
58c3a5ccd